rust/clients/bridge/README.md

```diff
@@ -0,0 +1,86 @@
+# qaul-cli
```

Comment:
Please write a specific README for the binary, that explains what the binary does and how to start it.